### PR TITLE
Robust walking policy for Stompy Pro

### PR DESCRIPTION
**Pull request by WT-MM**

### CODE REVIEW

File: sim/envs/humanoids/stompypro_config.py [hunk 1]
```diff
@@ -282,7 +228,7 @@ class runner:
         policy_class_name = "ActorCritic"
         algorithm_class_name = "PPO"
         num_steps_per_env = 60  # per iteration
-        max_iterations = 3001  # number of policy updates
+        max_iterations = 8001  # number of policy updates
```
Comment (budzianowski) [inline sim/envs/humanoids/stompypro_config.py:231]: keep 3000

File: sim/resources/stompypro/joints.py [hunk 2]
```diff
@@ -185,31 +195,42 @@ def stiffness(cls) -> Dict[str, float]:
...
 
-    # pos_limits
-    @classmethod
-    def effort(cls) -> Dict[str, float]:
-        return {}
 
-    # vel_limits
     @classmethod
-    def velocity(cls) -> Dict[str, float]:
+    def damping_mujoco(cls) -> Dict[str, float]:
```
Comment (budzianowski) [inline sim/resources/stompypro/joints.py:207]: remove damping mujoco

**Review by budzianowski — COMMENTED** [review]
Please retrain after you create robot_fixed with updated efforts parameters so that stiffness and effort values are the same.

File: sim/envs/humanoids/stompypro_config.py [hunk 3]
```diff
@@ -143,11 +125,9 @@ class domain_rand(LeggedRobotCfg.domain_rand):
         added_mass_range = [-1.0, 1.0]
         push_robots = True
         push_interval_s = 4
-        max_push_vel_xy = 0.3  # 0.2
+        max_push_vel_xy = 0.3 # 0.2
         max_push_ang_vel = 0.4
-        # dynamic randomization
-        action_delay = 0.5
```
[the remark this answers is not in the document]
Comment (WT-MM) [reply sim/envs/humanoids/stompypro_config.py:149]: action_delay isn't used anywhere anymore and dynamic_randomization is a diff name for action_noise I think

[the remark this answers is not in the document]
Comment (budzianowski) [reply sim/envs/humanoids/stompypro_config.py:149]: I introduced it  recently - please revert back to this setup

**Pull request merged**